style.css

```diff
@@ -0,0 +1,526 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.wrapper {
+  max-width: 930px;
+  min-width: 320px;
+  font-family: 'GothamPro', serif;
+  margin: 0 auto;
+  box-sizing: border-box;
+}
+
+/* header */
+
+.header {
+  width: 100%;
+  height: 63px;
+  display: grid;
+  flex-direction: row;
+  grid-template-columns: 0.7fr 4fr 1.5fr 0.6fr 0.6fr 0.3fr;
+  margin-bottom: 33px;
+  padding-top: 24px;
+  padding-bottom: 8px;
+}
+
+.header__logo {
+  display: flex;
+  align-items: center;
+}
+
+.header__logo-image {
+  width: 55px;
+  height: 55px;
+}
+
+.header__logo-image_adaptive {
+  display: none;
+}
+
+.header__menu {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-right: 140px;
+}
+
+.header__menu-bar {
+  display: none;
+}
+
+.menu-bar {
+  display: none;
+}
+
+.link {
+  display: inline-block;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: bold;
+  line-height: 11px;
+}
+
+.link__white {
+	color: #fff;
+}
+
+.header__search {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  width: 160px;
+}
+
+.search-input {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  border: none;
+  border-bottom: 1px solid #000;
+  padding-bottom: 20px;
+}
+
+.search-icon {
+  display: flex;
+  position: absolute;
+  left: 148px;
+  align-items: center;
+  justify-content: center;
+  width: 14px;
+  height: 14px;
+}
+
+.header__user-block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.header__cart {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.cart-icon {
+  margin-left: 5px;
+}
+
+.header__language-block {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
+
+.header__language-image {
+  width: 20px;
+  height: 20px;
+}
+
+/* top-content */
+
+.top-content {
+  background-image: url(img/header-img.png);
+  display: block;
+  width: 100%;
+  height: 253px;
+  margin-bottom: 67px;
+}
+
+.top-content__block {
+  display: flex;
+  flex-direction: column;
+  width: 209px;
+  padding-left: 54px;
+  padding-top: 78px;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.top-content__block-link {
+	display: flex;
+	justify-content: flex-start;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)